app/src/main/java/com/kkolontay/theworld/flow/Flows.kt

```diff
@@ -0,0 +1,25 @@
+package com.kkolontay.theworld.flow
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.cancellable
+import kotlinx.coroutines.flow.flow
+
+object Flows {
+    var counter = 0
+
+    suspend  fun refresh() {
```

Comment:
Note: There's a warning here that the `suspend` modifier is redundant.  It can be removed.

```suggestion
    fun refresh() {
```